main.py

```diff
@@ -0,0 +1,90 @@
+"""
+This script monitors specified stocks, compares their prices with historical averages,
+and sends email notifications when certain conditions are met.
+"""
+
+#Import the necessary libraries and modules
+import yfinance as yf
+import smtplib
+import schedule
+import time
+import os
+from forex_python.converter import CurrencyRates
+from dotenv import load_dotenv
+from statistics import mean
+
+# Load environment variable from .env file
+load_dotenv()
+
+# Initialize currency converter
+currency_converter = CurrencyRates()
+
+# Define the stocks to monitor and the email details
+stocks_to_monitor = ["AAPL", "MSFT", "GOOGL", "TSLA", "NKE"]
+previous_prices = {ticker: 0 for ticker in stocks_to_monitor}
```

Comment:
is that the initial dictionary? Will this be updated throughout the running of the code?